### PR TITLE
[PDE-1820] Update Zapier Platform CLI links

### PR DESCRIPTION
**Pull request by kimgault**
## Description
Zapier Platform CLI links were pointing to an archived repo, an outdated wiki or to a file that was deleted. Updates were made to these links.

https://github.com/zapier/zapier-platform/wiki was updated too. Tutorial now points to a Visual Builder tutorial and the example apps links were updated.

LMK if I missed anything or if anything should be reverted!

## Issue

https://zapierorg.atlassian.net/browse/PDE-1820

### CODE REVIEW

File: docs/_cli_docs/docs.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ You can develop using any version of Node you'd like, but your eventual code mus
 
 To ensure stability for our users, we strongly encourage you run tests on `v12` sometime before your code reaches users. This can be done multiple ways.
 
-Firstly, by using a CI tool (like [Travis CI](https://travis-ci.org/) or [Circle CI](https://circleci.com/), which are free for open source projects). We provide a sample [.travis.yml](https://github.com/zapier/zapier-platform/blob/master/example-apps/minimal/.travis.yml) file in our template apps to get you started.
```
Comment (kimgault) [inline docs/_cli_docs/docs.md:74]: Pointing this out for approval: Minimal template no longer has a travis yml file, so I pointed it to the create template's file instead.

File: docs/_cli_docs/docs.md [hunk 2]
```diff
@@ -102,7 +102,7 @@ cd example-app
 npm install
 ```
 
-> Note: there are plenty of templates & example apps to choose from! [View all Example Apps here.](https://github.com/zapier/zapier-platform/tree/master/example-apps)
+> Note: there are plenty of templates & example apps to choose from! [View all Example Apps here.](https://github.com/zapier/zapier-platform/wiki/Example-Apps)
```
Comment (stevelikesmusic) [inline docs/_cli_docs/docs.md:105]: Not for this PR, but I wonder if the wiki should really be a README for the [example-apps package](https://github.com/zapier/zapier-platform/tree/master/example-apps)

**Review by stevelikesmusic — APPROVED** [review]
Ah the Links to Nowhere 😂 Thanks for getting these @kimgault!

File: docs/_cli_docs/docs.md (hunk 2 above)
Comment (kimgault) [reply docs/_cli_docs/docs.md:105]: Makes sense! I created this Jira ticket for the backlog: https://zapierorg.atlassian.net/browse/PDE-2026

**Pull request merged**